egs/swbd/s5c/local/reverberate/prep_sim_rirs.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# Copyright 2016  Tom Ko
+# Apache 2.0
+# script to simulate room impulse responses (RIRs)
+
+sampling_rate=8000
```

Comment:
Are you talking about whether matlab itself is installed?
Definitely there should be a check for that.
I'm very unhappy with having a dependency on matlab at all-- many people can't afford it.    But I suppose it won't be very easy to get rid of this dependency.  Even ensuring octave compatibility would be better than relying on matlab itself.